### DISCUSSION TITLE
IBM Aptiva ROM Files

### DISCUSSION THREAD

**Discussion by RyanH-Nostalgia** [86Box/86Box#2248] [General]
Hi there, my name is Ryan Hawley (Am al also youtube user PB70CDOEM) I have a few IBM Aptiva computers that I would like to copy hardware related content to ROM files with. I am not sure how else to describe this so I will do it this way,

Here is a list of things I would like to create ROM files of systems that I have and to share and have added to 86Box,

* IBM Aptiva model 2168-M71 (System board bios)
* IBM Aptiva model 2159-S7H (System board bios)
* IBM Mwave 20D Sound/modem Combo ISA card (Used in the 2144-Mxx/Axx and 2168-Mxx/Axx Aptiva Models and the 2176/2134-Cxx Aptiva Models as well.)
* IBM Mwave 20E Sound/modem Combo ISA card (Used in the 2159-Sxx Aptiva Models)

I also want to point out that the Aptiva Systems I have also have a Hardware based feature called Rapid Resume, it's a power based feature that the IBM Company added in. there are a few different versions of it.

I also have an IBM PS/1 Model 2155 and an IBM PS/1 Model 2168 which have an earlier version of the Rapid Resume there have not yet been any of those added to 86Box and I am happy to make ROM Image files of those systems as well.

here is a link to a video that I had made about this subject originally for PCem and still is if it is still going on,
https://www.youtube.com/watch?v=QoBP-cPN2zo

if there is any program that someone here can provide to me so I can make these ROM files from my IBM Aptiva Systems and share them online, please let me know.

I also want to point out that I am new to this, and I am also not a programmer and do not know how any of this stuff works.

**Comment by ManoDaSilva** [comment]
If that helps, here is a BIOS dump of an Aptiva 2176 (french edition)
[J12BX71W4U4-Dump-Chk01FEF700.zip](https://github.com/user-attachments/files/17632449/J12BX71W4U4-Dump-Chk01FEF700.zip)

**Comment by xog** [comment]
Where you able to dump the ROM form the machines?

**Reply by RyanH-Nostalgia** [reply]
Hi there, it has been quite some time since I posted about this on here and I have not forgotten about my post I made ether, there has just been quite a bit of things going on in my life. That being said, to answer your question, I have not been able to do any ROM dumps of my IBM Aptiva systems or the Mwave sound and modem cards because I do not have a program or utility to do so and I posted on here to see if someone could provide a utility of some sort so that I can do so, I am still up for doing ROM dumps of my IBM Aptiva systems as well as the IBM PS/1 systems that I have, I would love to be able to emulate those systems on my modern PCs.

**Comment by xog** [comment]
I will see if i can help you figure out how to do it since im also intresed in beeing able to emulate the Aptiva systems.

**Comment by errorxxx451xxx** [comment]
i need only mwave driver

**Reply by RyanH-Nostalgia** [reply]
Hi there, Sorry it took me a while to reply. I have uploaded 2 different CDs with different sets of IBM Mwave Drivers to my Archive.org account and I have a youTube Channle where I showed how to install them on my PB70CDOEM YouTube channle.

https://archive.org/details/ibm-aptiva-service-update-cd-srvcd-0-l

https://archive.org/details/TECHUPCD
